client/src/components/standing_table/standingData.ts

```diff
@@ -0,0 +1,61 @@
+/* Home page Data */
+export const homeStandingHeader = ['Pos', 'Club', 'Pl', 'GD', 'Pts'];
+
+export interface HomeStanding {
+  name: JSX.Element;
+  played: number;
+  GD: number;
+  points: number;
+}
```

Comment:
is this getting used somewhere?

Reply:
Yes. 
It is used in `HomeContent.tsx` to show the table on the homepage.